java-security/src/test/java/com/sap/cloud/security/adapter/spring/SpringOAuth2ClientSecurityContextTest.java

```diff
@@ -0,0 +1,62 @@
+package com.sap.cloud.security.adapter.spring;
+
+import com.sap.cloud.security.token.SapIdToken;
+import com.sap.cloud.security.token.Token;
+import org.apache.commons.io.IOUtils;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.oauth2.core.oidc.OidcIdToken;
+import org.springframework.security.oauth2.core.oidc.user.DefaultOidcUser;
+
+import java.io.IOException;
+import java.time.Instant;
+
+import static com.sap.cloud.security.config.Service.IAS;
+import static com.sap.cloud.security.token.TokenClaims.SUBJECT;
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static java.util.Collections.singleton;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class SpringOAuth2ClientSecurityContextTest {
+	SapIdToken sapIdToken;
+
+	@Before
```

Comment:
I would suggest for all new test classes that we create to use only junit5 going forward :interrobang: